### ISSUE TITLE
Chart Colours only uses first array item

### ISSUE THREAD

**Issue by eldo-xy** [RinMinase/ng-charts#3]
If you provide an array of colours to be used, only the first item is used and colours all segments/lines with that colour.

## Expected Behavior
Each segment should have the defined colour from the array

## Current Behavior
Currently all segments get the same colour

## Steps to Reproduce
A slight modification to one of the examples in the readme demonstrates the issue. Three colours given in the array, #f00, #0f0, #00f. The pie chart is then renders with all segments set as #f00

[https://stackblitz.com/edit/ng-charts-pie-3twyyr](https://stackblitz.com/edit/ng-charts-pie-3twyyr)

**Comment by RinMinase** [comment]
Hello @eldo-xy, 😁
Thank you for pointing it out. I seem to have forgotten to include it in the examples.

I have modified the example for the [pie chart](https://stackblitz.com/edit/ng-charts-pie) in the readme file.
The chart's `[colors]` are formatted using an `Array<object>`.

```typescript
data = [300, 500, 100];
colors = [{
    backgroundColor: [
        'red',
        '#0F0',
        'rgba(41, 182, 246,0.75)'
    ],
    borderColor: [
        'rgb(250,120,100)',
        'green',
        '#0086c3'
    ]
}];
```
```html
<canvas baseChart chartType="pie" [data]="data" [colors]="colors"></canvas>
```

With regards to how the `[color]` should be an array of objects, yet only uses the first element, I'll have a look at it sometime on how I can improve the implementation of this.

_Feel free to reopen this issue if it still persists._

**Comment by RinMinase** [comment]
With regards to how the `[color]` should be an array of objects, yet only uses the first element, I'll have a look at it sometime on how I can improve this.

**Comment by eldo-xy** [comment]
Thanks, that resolves my issue.

This line in the readme is what lead me down this route
`colors (Color[]) - data colors, will use default and|or random colors if not specified (see below)`
This suggests it's an array of Color, not a Color whose properties are an array of the color values. I think a modified demo explains the intended implementation though.

**Comment by RinMinase** [comment]
> This line in the readme is what lead me down this route

My bad. I didn't notice that.
I'll have that changed. 👍🏻

**Comment by RinMinase** [comment]
This has been amended in **1fd540c** and **a9a3702**, released in **v3.0.0**
